pkgs/by-name/lu/lux-cli/package.nix

```diff
@@ -0,0 +1,116 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  libgit2,
+  gnupg,
+  libgpg-error,
+  gpgme,
+  darwin,
+  installShellFiles,
+  luajit,
+  makeWrapper,
+  symlinkJoin,
+  lua51Packages,
+  lua52Packages,
+  lua53Packages,
+  lua54Packages,
+  nix,
+  versionCheckHook,
+}:
+let
+  lux-lua-bundle = symlinkJoin {
+    name = "lux-lua-bundle";
+    paths = [
+      lua51Packages.lux-lua
+      lua52Packages.lux-lua
+      lua53Packages.lux-lua
+      lua54Packages.lux-lua
+    ];
+  };
+in
```

Comment:
This is kinda rough for build time. We end up having to build this derivation multiple times :S

Reply:
Yep. This is a temporary solution, until https://github.com/nvim-neorocks/lux/issues/655 has been implemented.
I left a comment about that in the `postFixup` phase.